backend/stakepoold/userdata/userdata.go

```diff
@@ -16,7 +16,7 @@ type DBConfig struct {
 
 // UserData stores the current snapshot of the user voting config.
 type UserData struct {
-	sync.RWMutex
+	mtx              sync.RWMutex
```

Comment:
That one has to stay -- gob encoder that is used for the cache will choke otherwise.